packages/paste-design-tokens/tokens/aliases/border-radius.yml

```diff
@@ -4,3 +4,4 @@ aliases:
   border-radius-20: "4px"
   border-radius-30: "8px"
   border-radius-circle: "50%"
+  border-radius-pill: "100px"
```

Comment:
question: is this the best name for this token? It's called pill, used on Badge, but not on Pill.

question: where did 100px come from as a value?

Reply:
This is a question for @twilio-labs/design-systems-pd as well; the name and value are from the spec.

Reply:
Ah, yes, this name and value were a suggestion, but I don't know what the perfect implementation is. The border radius is used to make a pill shape (so, following the logic of border-radius-circle), but no, it is not used in Pill right now (Pills should probably be shaped like pills but...we can tackle that at a later date haha). Very open to other token name suggestions.

I don't know what the pixel value should be - I just tossed a random high pixel value out as a suggestion but I have no attachment to 100px.

Reply:
I can buy pill as a name based on shape.

100px _feels_ like a magic number, but we can always change it later to something more specific